Setup Vagrant

adhikaribiraj opened the pull request.

Commit:
Setup Vagrant

StoneyJackson commented:
Did you mean to make a PR to LibreFoodPantry (instead of to StoneyJackson)?

KevinTran000 committed:
- Resolved ujson dependency issue.

KevinTran000 committed:
- Modified Vagrantfile to install other dev. tools
-  Also print statements for easier output reading.

KevinTran000 committed:
Merge branch 'master' of https://github.com/StoneyJackson/BEAR-Necessities-Market into setup-vagrant

KevinTran000 committed:
Slight refactor

KevinTran000 committed:
Slight refactor

StefieNguyen committed:
updated README to setup project with Vagrant

StefieNguyen committed:
Merge branch 'setup-vagrant' of https://github.com/StoneyJackson/BEAR-Necessities-Market into setup-vagrant

KevinTran000 committed:
- Readme tweaks.

- Added links to readme
- Standardized a few wordings
- Wrapped command in code backticks
- Reword a section to make more apparent that a section terminal is required.

KevinTran000 committed:
- Readme tweaks

- Clarifying a section

KevinTran000 committed:
- Readme tweaks

- Standardizing section headers wih colons.

KevinTran000 committed:
- Readme tweaks

- Fixed typo
- Fixed capitalization errors

KevinTran000 committed:
Merge branch 'master' of https://github.com/LibreFoodPantry/BEAR-Necessities-Market into setup-vagrant

KevinTran000 committed:
- Readme tweaks

- Added `--host=0.0.0.0` switch to `flask run` instruction

Commit:
Remove duplicate getting started guide

Commit:
Delete the getting started file

KevinTran000 committed:
Merge branch 'master' of https://github.com/LibreFoodPantry/BEAR-Necessities-Market into setup-vagrant

ndion617 commented:
On windows having an issue with `$flask db upgrade`
`vagrant@vagrant:~$ export FLASK_APP=manage.py`
`vagrant@vagrant:~$ export SENDGRID_API_KEY="RANDOMKEY"`   
 `vagrant@vagrant:~$ export SENDGRID_DEFAULT_FROM="example@example.com"`

`vagrant@vagrant:~$ flask db upgrade`                                                      
`Usage: flask db upgrade [OPTIONS] [REVISION] `                                                                                                                                       `Error: Could not import "manage".`

KevinTran000 committed:
- Readme Tweaks

- Added Minimum System Requirements
- Condensed some instructions for clarity
- Clarified terminal location
- Added instructions for how to shutdown, suspend, and boot up the vagrant box.
